server/services/roService.ts

```diff
@@ -102,3 +121,9 @@ function toResponsibleOfficer(offenderNumber: string, offenderManager: Community
     probationAreaDescription: probationArea.description,
   }
 }
+
+function absentDatesToCompare(booking): Boolean {
```

Comment:
**suggestion(non-blocking):** like the name above `hasNoDatesToCompare` could use that here: `hasNoDatesToCompare`

Reply:
Changing it to `noDatesToCompare` to avoid `'hasNoDatesToCompare' is already declared in the upper scope` error